mmv1/products/privateca/api.yaml

```diff
@@ -344,12 +476,11 @@ objects:
   # Certificate
   - !ruby/object:Api::Resource
     name: 'Certificate'
-    base_url: 'projects/{{project}}/locations/{{location}}/certificateAuthorities/{{certificate_authority}}/certificates'
-    create_url: 'projects/{{project}}/locations/{{location}}/certificateAuthorities/{{certificate_authority}}/certificates?certificateId={{name}}'
-    self_link: 'projects/{{project}}/locations/{{location}}/certificateAuthorities/{{certificate_authority}}/certificates/{{name}}'
-    min_version: beta
-    delete_url: 'projects/{{project}}/locations/{{location}}/certificateAuthorities/{{certificate_authority}}/certificates/{{name}}:revoke'
-    delete_verb: :POST    
+    base_url: 'projects/{{project}}/locations/{{location}}/caPools/{{pool}}//certificates'
+    create_url: 'projects/{{project}}/locations/{{location}}/caPools/{{pool}}/certificates?certificateId={{name}}'
+    self_link: 'projects/{{project}}/locations/{{location}}/caPools/{{pool}}/certificates/{{name}}'
```

Comment:
nit: This is the default `self_link` and could be omitted, right?

Reply:
Ha, looks like `base_url` had two `/` characters, which probably made this necessary. Good catch!